app/models/cart.rb

```diff
@@ -17,7 +17,11 @@ def minus_item(item_id)
   end
 
   def sum
-    drinks.map {|total|total.sum}.reduce(:+)
+    drinks.map {|drink| drink.sum}.reduce(:+)
+  end
+
+  def total_quantity
+    drinks.map {|drink| drink.quantity}.reduce(:+)
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Pass `&:quantity` as an argument to `map` instead of a block.